src/styles/blocks/_watch.scss

```diff
@@ -0,0 +1,49 @@
+@import "./utils/extends.scss";
+
+.watch {
+  @include square($clock-size);
+
+  position: relative;
+
+  border-radius: 50%;
+
+  animation-name: scale;
+  animation-duration: 60s;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+
```

Comment:
I'm not sure if this is correct, but you can write it in one line
`animation: scale 60s infinite linear`